Cargo.toml

```diff
@@ -7,6 +7,12 @@ authors = ["The New Relic Agent Control Team"]
 publish = false
 license-file = "./LICENSE.md"
 
+[workspace.package]
+authors = ["The New Relic Agent Control Team"]
+edition = "2024"
+publish = false
+rust-version = "1.85.0"
+license-file = "./LICENSE.md"
```

Comment:
The repository is not a workspace. Why did we add that?
Why don't we add the [rust-version](https://doc.rust-lang.org/cargo/reference/rust-version.html) directly under the `package` section?

By the way, is that really the minimum supported version or did we add it just to use it in the pipeline? If that's the case, maybe we should hardcode a version or use the latest version in the pipeline and that's it.

Reply:
I do not know what I was thinking, thanks for pointing out